connect/runtime/src/main/java/org/apache/kafka/connect/util/ConnectUtils.java

```diff
@@ -65,4 +70,15 @@ static String lookupKafkaClusterId(Admin adminClient) {
                                        + "Check worker's broker connection and security properties.", e);
         }
     }
+
+    public static void addMetricsContextProperties(Map<String, Object> prop, WorkerConfig config, String clusterId) {
+        //add all properties start with "metrics.context"
+        for (String key : config.originals().keySet()) {
+            if (key.startsWith(KafkaMetricsContext.METRICS_CONTEXT_PREFIX)) {
+                prop.put(key, config.originals().get(key));
+            }
+        }
```

Comment:
this code is the same as the one we have in KafkaMetricsContext, so if we move it into a separate utility method as I suggested, we can reuse it here and not repeat ourselves.